bokeh/io.py

```diff
@@ -37,6 +37,8 @@
 from .util.notebook import load_notebook, publish_display_data, get_comms
 from .util.string import decode_utf8
 from .util.serialization import make_id
+from .resources import INLINE
+from .core.templates import ZEPPELIN_FILE
```

Comment:
Right we could either have a `bokeh.util.notebook` that has methods for all notebooks, e.g. `load_jupyter` and `load_zeppelin`, or we could split into two modules `bokeh.util.jupyter` and ``bokeh.util.zeppelin` There are pluses and minuses to both approaches, I don't have a strong opinion on which way we go. 

Regarding the resources, it's also up to you, I was only intending to give some historical context. Given that *some* way of loading resources is always necessary, I guess my natural inclination is to pick the "correct" way from the start. But if it is easier to bootstrap the work (or if you think Zeppelin users would actually prefer inline resources) then certainly that's fine.